mesonbuild/mtest.py

```diff
@@ -346,20 +364,23 @@ def make_tap(test, returncode, duration, stdo, stde, cmd):
             else:
                 res = TestResult.FAIL if failed else TestResult.OK
 
-        return TestRun(test, res, returncode, duration, stdo, stde, cmd)
+        return cls(test, test_env, res, returncode, duration, stdo, stde, cmd)
 
-    def __init__(self, test, res, returncode, duration, stdo, stde, cmd):
+    def __init__(self, test: 'TestSerialisation', test_env: typing.Dict[str, str],
+                res: TestResult, returncode: int, duration: float,
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5395#issue-4079292))

<sub>posted by Sider</sub>